test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.0.2"
```

Comment:
I was seeing the following errors in the Windows builds:

11:53:28 The specified framework 'Microsoft.NETCore.App', version '1.0.0' was not found.
11:53:28   - Check application dependencies and target a framework version installed at:
11:53:28       D:\j\workspace\release_windo---d8901c9b\.dotnet_stage0\Windows\x64\shared\Microsoft.NETCore.App
11:53:28   - The following versions are installed:
11:53:28       1.0.2-servicing-004528-00
11:53:28   - Alternatively, install the framework version '1.0.0'. 

@brthor suggested bumping the version of that package, he can probably tell you more. I'm still tinkering.